src/tests/webkit/WebKit_Performance_010.test.js

```diff
@@ -5,7 +5,7 @@ export default {
   ...{
     context: {
       minFPS: 30,
-      url: 'http://helloracer.com/racer-s/',
+      url: 'http://cdn.metrological.com/static/eme-v3-clean.html',
```

Comment:
You can drop this if the race car isnt available anymore, reusing the eme racecar is not the same.

Reply:
Dropping(Removing) this test as  the hello racer isn't available